src/Web/Program.cs

```diff
@@ -73,6 +75,8 @@
                 Name = "Authorization",
                 Type = SecuritySchemeType.ApiKey
             });
+        var filePath = Path.Combine(AppContext.BaseDirectory, "Web.xml");
+        c.IncludeXmlComments(filePath);

```

Comment:
Hvilken fil er dette og hvorfor trenger man den?

Reply:
Se endringer i src/Web/Web.csproj 

Jeg inkluderte denne sånn at kommentaren i summary kommer med i Swagger. Tenkte at det var lurt, så jeg kunne forklare hva endepunktene gjør direkte fra Swagger.

Reply:
![bilde](https://github.com/varianter/chewbacca/assets/25222071/61a117dd-38bd-41ce-a8dd-d26378344c48)

Reply:
Skjønner! Sjekka litt og så eksemplet her: https://github.com/domaindrivendev/Swashbuckle.AspNetCore#include-descriptions-from-xml-comments at bruker `<GenerateDocumentationFile>true</GenerateDocumentationFile>` i csproj-fila. Er kanskje bedre for oss og?

Reply:
Gjort. Rider gjorde det automatisk som med debug/release men det fungerte også sånn.